### ISSUE TITLE
[BUG] scrollIntoView for TableVirutoso with fixedHeader buggy

### ISSUE THREAD

**Issue by mortmoe** [petyosi/react-virtuoso#745]
**Describe the bug**
When implementing keyboard navigation using the examples in docs combined with table it seems like scrollIntoView doesn't take the height of the header into consideration, failing to scroll the new selected element into view (or scrolling it just partially into view)

**Reproduction**
(https://codesandbox.io/s/sandpack-project-forked-s0ivuu)

**To Reproduce**
Quick codesandbox. Click the header then press arrow down. For some reason you need to click the header between each arrow down, looses focus or something like that, but it should demonstrate the issue. When you get to the bottom you see that the new selected element is not fully scrolled into view. Remove the fixed header and it will work as intended.

**Expected behavior**
Should scroll the whole item into view

**Desktop (please complete the following information):**
 - OS: Linux.
 - Browser Chrome

**Comment by mortmoe** [comment]
Thanks for the quick fix :D It's better now, but still there is a small issue. It seems like an item is considered "in view" if the top of the item is within view, not the whole item (or actually top AND bottom of the item). I'm attaching a screencast to show the issue. Notice that the bottom item is barely visible at the beginning, everything works fine as long as the top of the item you are scrolling is not visible :) (This is virtuoso 2.18.0)
[Uploading Screencast from 07. sep. 2022 kl. 09.17 +0200.webm…]()

**Comment by petyosi** [comment]
That's a nice detail which totally went over my head when I did that feature. Certainly a different issue, though.
